backend/api/publishing.py

```
@@ -237,6 +240,307 @@ async def get_draft_scenarios(
+        
+        if persona_upload_tasks:
+            persona_results = await asyncio.gather(*persona_upload_tasks, return_exceptions=True)
+        
+        if scene_upload_tasks:
+            scene_results = await asyncio.gather(*scene_upload_tasks, return_exceptions=True)
+        
+        # Process persona upload results
+        for i, result in enumerate(persona_results):
+            persona, temp_url = persona_task_map[i]
+            if isinstance(result, Exception):
+                debug_log(f"[IMAGE_STORAGE] Persona {persona.name} (ID {persona.id}): Upload failed with exception, keeping temporary URL: {str(result)}")
```

Ruff (0.14.3)

Use explicit conversion flag

Replace with conversion flag

(RUF010)